public/test/utils/regions/packages.js

```diff
@@ -0,0 +1,70 @@
+import {type} from 'test/utils/dom-nodes';
+import $ from 'jquery';
+import textInside from 'test/utils/text-inside';
+import * as wait from 'test/utils/wait';
+
+class Package {
+    constructor(dom) {
+        this.dom = dom;
+    }
+
+    createNewPackage(name) {
+        $(this.dom.querySelector('.package-create')).click();
+        type($('.createPackageName_input', this.dom), name);
+        $(this.dom.querySelector('.createPackageSave')).click();
+        return wait.event('find:package');
```

Comment:
I'm emitting a package creation event now after the package creation and this function now waits for that event.